packages/react-instantsearch-hooks-dom/src/widgets/PoweredBy.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { usePoweredBy } from 'react-instantsearch-hooks';
+
+import { PoweredBy as PoweredByUiComponent } from '../ui/PoweredBy';
+
+import type { ClassNames } from '../types';
+import type { PoweredByProps as PoweredByUiComponentProps } from '../ui/PoweredBy';
+
+export type PoweredByProps = Omit<ClassNames<PoweredByUiComponentProps>, 'url'>;
+
+export function PoweredBy({ classNames = {}, ...props }: PoweredByProps) {
+  const { url } = usePoweredBy();
+
+  return <PoweredByUiComponent {...props} url={url} classNames={classNames} />;
+}
```

Comment:
In conjunction with the other suggestions, we could now remove the defaulted `classNames` prop here and let it be passed to the UI component with `...props`:

```suggestion
export function PoweredBy(props: PoweredByProps) {
  const { url } = usePoweredBy();

  return <PoweredByUiComponent {...props} url={url} />;
}
```

Reply:
Addressed in e84cf31a.